tests/test_measures.py

```diff
@@ -219,25 +240,33 @@ def test_log_likelihood_with_zeros(zero_dataframe):
 ######
 
 @pytest.mark.hypergeometric_likelihood
-def test_hypergeometric_likelihood(sample_dataframe):
+def test_hypergeometric_likelihood_single(sample_dataframe):
+    import association_measures.frequencies as fq
     df = sample_dataframe
+    df = fq.observed_frequencies(df)
+    df = df.join(fq.expected_frequencies(df))
+    df_ams = am.hypergeometric_likelihood(df)
+    assert df_ams[0] == 5.776904234533874e-14
 
-    df = sample_dataframe
-    df_ams = am.calculate_measures(df, ['hypergeometric_likelihood'])
-    assert df_ams['hypergeometric_likelihood'][0] == 5.776904234533874e-14
 
+# @pytest.mark.hypergeometric_likelihood
+# def test_hypergeometric_likelihood(sample_dataframe):
+#     df = sample_dataframe
+#     df_ams = am.calculate_measures(df, ['hypergeometric_likelihood'])
+#     assert df_ams['hypergeometric_likelihood'][0] == 5.776904234533874e-14
```

Comment:
Why are all the tests disabled?

Reply:
the hypergeometric likelihood is disabled in the standard interface `calculate_measures`. These tests thus don't work right now.

Reply:
I think a better option would be a skip mark and a comment. https://docs.pytest.org/en/stable/skipping.html

Reply:
this is very true. changed in latest commit.